### PR TITLE
Upgrade Label Studio to 1.4.0

### PR DESCRIPTION
**Pull request by anayden**

### CODE REVIEW

File: .neuro/live.yml [hunk 1]
```diff
@@ -191,29 +193,29 @@ jobs:
...
-        start --use-gevent --no-browser ${LABEL_STUDIO_PROJECT} 
+        -- \
+        start dogs --port ${LS_PORT} --no-browser
 
       # Push labeling completions (label studio-specific files), images and annotation results
       pachctl start commit ${PACHY_REPO}@master
-      pachctl put file -r ${PACHY_REPO}@master:/ls-completions/ -f ${COMPLETIONS} | tee
       pachctl put file -r ${PACHY_REPO}@master:data/Images/ -f ${DATA_PATH} | tee
       pachctl put file    ${PACHY_REPO}@master:data/result.json -f ${PROJECT}/data/result.json | tee
+      pachctl put file    ${PACHY_REPO}@master:data/label_studio.sqlite3 -f /label-studio/data/label_studio.sqlite3 | tee
```
Comment (YevheniiSemendiak) [inline .neuro/live.yml:216]: completions are now stored in SQLite? (just curious)

File: label_studio/extend_dataset.py [hunk 2]
```diff
@@ -1,63 +1,51 @@
...
-                    "choice": breed,
-                }
-            )
-            i += 1
-    if not args.skip_annotation_update:
-        with cur_dataset_descr.open("w") as fd:
-            json.dump(cur_dataset_descr_d, fd, indent=2, sort_keys=True)
+    # Select and copy new images
+    new_breed_images = available_breed_images[:args.nmber_of_imgs]
+    random.shuffle(new_breed_images)
```
Comment (YevheniiSemendiak) [inline label_studio/extend_dataset.py]: Maybe,
```suggestion
    random.shuffle(available_breed_images)
    new_breed_images = available_breed_images[:args.nmber_of_imgs]
```

File: README.md [hunk 3]
```diff
@@ -42,6 +42,12 @@ Take the full token from the command's output and store it in a secret:
 neuro secret add platform-config FULL_TOKEN_FROM_OUTPUT
 ```
 
+Create secret for Label Studio token:
```
Comment (YevheniiSemendiak) [inline README.md:45]: Pls, add also a note regarding `secret:pachyderm-token`

**Review by YevheniiSemendiak — COMMENTED** [review]
Great job and simplifications in overall LS management, thank you, Lesha!

Feels like sooner or later we will be able to run LS as managed service ;)
Left one request, suggestion, and question out of curiosity. Only "pachy-token" seem to be terra incognita for me :)

File: .neuro/live.yml (hunk 1 above)
Comment (anayden) [reply .neuro/live.yml:216]: yes

File: README.md (hunk 3 above)
Comment (anayden) [reply README.md:45]: Oh, I forgot to remove all trace of `secret:pachyderm-token`. It's not used any more.

**Review by YevheniiSemendiak — APPROVED** [review]
LGTM! Thank you, mate! <3

**Pull request merged**